charts/dify/README.md

```diff
@@ -159,11 +174,30 @@ global:
         key: S3_SECRET_KEY
 ```
 
+#### Google Cloud Storage
+
+1. set the `minio.embedded` to `false`
+2. inject connection info with `global.extraBackendEnvs`
+
+```yaml
+global:
+  extraBackendEnvs:
+  - name: STORAGE_TYPE
+    value: "google-storage"
```

Comment:
These part needs to be changed to the latest syntax

Reply:
Yeap, sorry for that.